src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmssExtension.md

```diff
@@ -30,6 +30,24 @@ The **Remove-AzureRmVmssExtension** cmdlet removes an extension from the Virtual
 
 ## EXAMPLES
 
+### Example 1: Remove a VMSS extension
+```
+PS C:\> $vmss = Get-AzureRmVmss -ResourceGroupName $RGName -VMScaleSetName $vmssName 
+PS C:\> Remove-AzureRmVmssExtension -VirtualMachineScaleSet $vmss -Name $vmssExtensionName
```

Comment:
@cloudmelon the `-ResourceGroupName` parameter is mandatory for this cmdlet, so you'll need to make sure to include it

Reply:
https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/remove-azurermvmssextension?view=azurermps-6.4.0

Actually the resource group name is used in 1st command to get the vmss as a object then we'll use it in passing as object parametter in Remove-AzureRmVmssExtension so it is not mandatory as mentionned in the documentation.  Thanks for verifying it with me :)

Reply:
@cloudmelon whoops, was looking at the documentation for `Remove-AzureRmVmss`, which has a required `-ResourceGroupName` parameter 😁

Reply:
yes @cormacpayne that is for Remove-AzureRmVmss, this page is about Remove-AzureRmVmssExtension :)